src/js/components/form-elements/SelectField.jsx

```diff
@@ -0,0 +1,48 @@
+import _ from 'lodash';
+import React from 'react';
+import Select from 'react-select';
+
+import 'react-select/dist/react-select.css';
+
+import BaseReactField from './BaseReactField';
+
+const SelectField = (props) => {
+  const renderInput = (attributes) => {
+    const options = _.map(attributes.options, value => (typeof value === 'string' ? { value, label: value } : value));
+
+    const onChange = (selectValue) => {
+      if (attributes.multi) {
+        const values = _.map(selectValue, val => val.value);
+        attributes.onChange(values);
+      } else if (selectValue !== null && selectValue !== undefined) {
+        attributes.onChange(selectValue.value);
+      } else {
+        attributes.onChange(null);
+      }
+    };
+
+    const delimiter = attributes.delimiter || ';';
+
+    return (
+      <div className="col-md-4">
```

Comment:
there might be exceptions out of this, but let's see along the way. If so, we'll parametrize this later on.

Reply:
Yup this was about the css part. I think for now it looks alright with forms. We can extend capabilities and customizability of these components in future tickets.